### ISSUE TITLE
Add declaration files for TS or @types/simple-scrollbar

### ISSUE THREAD

**Issue by Beraliv** [buzinas/simple-scrollbar#50]
While adding import to TS file

```js
import SimpleScrollBar from 'simple-scrollbar';
```

The error is shown:
<img width="502" alt="screen shot 2018-10-02 at 12 28 45" src="https://user-images.githubusercontent.com/2991847/46340653-c2024200-c63e-11e8-82e6-1b7a47206067.png">

It would solve the problem

**Comment by Beraliv** [comment]
Steps to add types:

1. add link in `package.json`:

```json
{
  "types": "./types.d.ts",
}
```

2. add file in root with respective types:

```typescript
declare namespace SimpleScrollBar {
    export function initEl (element: Element): void;
    export function initAll (): void;
    export function moveBar (e: Event): void;
}

export default SimpleScrollBar
```

**Comment by Beraliv** [comment]
I've added types and need to check if they're correct
https://github.com/buzinas/simple-scrollbar/pull/51

**Comment by atilkan** [comment]
@Beraliv 
Also this.
`declare module "simple-scrollbar";`

**Comment by buzinas** [comment]
This was fixed by #51.
